docs/typescript.md

```diff
@@ -0,0 +1,53 @@
+# Typescript
+
+This guide shows you an example of how to develop a function in Typescript with
+the Functions Framework.
+
+1. Use [gts](https://github.com/google/gts) to configure Typescript.
+
+    ```sh
+    npx gts init
+    ```
+
+2. Install the required package(s):
+
+    ```sh
+    npm install @google-cloud/functions-framework
+    ```
+
+    In this example we are using an HTTP signature and require express
+    types:
+
+    ```sh
+    npm install @types/express -D
+    ```
+
+3. Add a `start` script to `package.json`, passing in the `--source` flag to
+   point to the compiled code directory:
```

Comment:
Where do we specify the compiled code directory? I think that step is missing from the instructions.

Reply:
similarly, this is added by `gts` in `tsconfig.json`, so we just mention that?

Reply:
No need to mention, thanks for telling me. Folks will figure out.